doc/website/blog/serverless-graphql-using-aws.md

```diff
@@ -0,0 +1,556 @@
+    "input": $util.toJson($context.args.input)
+  }
+}
+```
+
+```vtl title="Mutation.addTodo"
+{
+  "version" : "2017-02-28",
+  "operation": "Invoke",
+  "payload": {
+  	"action": "addTodo",
+    "input": $util.toJson($context.args.input)
+  }
+}
+```
+
+```vtl title="Mutation.removeTodo"
+{
+  "version" : "2017-02-28",
+  "operation": "Invoke",
+  "payload": {
+  	"action": "removeTodo",
+    "input": $util.toJson($context.args.input)
+  }
+}
+```
+
+The request mapping templates let us construct the event objects with which we invoke the Lambda functions.
+Through the `$context` object, we have access to the GraphQL request and the authentication session.
+In addition, it is possible to arange multiple resolvers sequentielly and reference the respectie outputs via the `$context` objct.
+In principle, it is also possible to define response mapping templates.
+However, in most cases it is suffucient to return the response object "as is".
+
+### Testing AppSync using the Query explorer
+
+The easiest way to test the API is to use the Query Explorer in AWS AppSync.
+Alternatively, one can register an API key in the settings of our AppSync API and use any standard GraphQL client.
+
+Let us first create a todo with the title `foo`:
+```graphql
+mutation MyMutation {
+  addTodo(input: {title: "foo"}) {
+    todo {
+      id
+      title
+    }
+  }
+}
+
+```
+<div style={{textAlign: 'center'}}>
+  <img alt="Screenshot of an executed addTodo Mutation using the AppSync Query Explorer" src="https://entgo.io/images/assets/appsync/todo-queries.png" />
+  <p style={{fontSize: 12}}>"addTodo" Mutation using the AppSync Query Explorer</p>
+</div>
+
+Requesting a list of the todos should return a single todo with title `foo`:
+
+```graphql
+query MyQuery {
+  todos {
+    title
+    id
+  }
+}
+
+```
+<div style={{textAlign: 'center'}}>
+  <img alt="Screenshot of an executed addTodo Mutation using the AppSync Query Explorer" src="https://entgo.io/images/assets/appsync/todo-queries-3.png" />
+  <p style={{fontSize: 12}}>"addTodo" Mutation using the AppSync Query Explorer</p>
+</div>
+
+Requesting the `foo` todo by id should work too:
+
+```graphql
+query MyQuery {
+  todo(id: "1") {
+    title
+    id
+  }
+}
+```
+<div style={{textAlign: 'center'}}>
+  <img alt="Screenshot of an executed addTodo Mutation using the AppSync Query Explorer" src="https://entgo.io/images/assets/appsync/todo-queries-4.png" />
+  <p style={{fontSize: 12}}>"addTodo" Mutation using the AppSync Query Explorer</p>
+</div>
+
+### Wrapping Up
+
+We successfully deployed a serverless GraphQL API for managing simple todos using AWS AppSync, AWS Lambda, and Ent.
+In particular, we provided step-by-step instructions on configuring AWS AppSync and AWS Lambda through the web console.
+Furthermore, we presented a proposal for how to structure our Go code.
+
+I successfully employed a similar but more complex setup for a B2B SaaS project I am working on part-time in a small team of two.
```

Comment:
```suggestion
I successfully deployed a similar but more complex setup for a B2B SaaS project I am working on part-time in a small team of two.
```

Reply:
I also removed this information because I found it not relevant enough.